examples/sztp/tests.sh

```diff
@@ -16,6 +16,7 @@ sleep 5
 # tests dhcp client
 docker-compose exec -T client cat /var/lib/dhclient/dhclient.leases
 docker-compose exec -T client cat /var/lib/dhclient/dhclient.leases | grep sztp-redirect-urls
+BOOTSTRAP=$(docker-compose exec -T client cat /var/lib/dhclient/dhclient.leases | grep sztp-redirect-urls | head -n 1 | awk '{print $3}' | tr -d '";')
```

Comment:
Just a note that these docker compose commands won't work if you install docker compose as a plugin, as the syntax in that case is `docker compose` instead of `docker-compose`. I have it installed as a plugin. Not worth fixing now, but I may come back with a fix for that FYI.